editor/editor_export.h

```diff
@@ -154,7 +154,9 @@ class EditorExportPlatform : public Reference {
 
 protected:
 	virtual void get_preset_features(const Ref<EditorExportPreset> &p_preset, List<String> *r_features) = 0;
+	bool exists_export_template(String template_file_name, String *err) const;
 	String find_export_template(String template_file_name) const;
+	String find_export_template(String template_file_name, String *err) const;
```

Comment:
Do we need to keep the two signatures? You could just have `err` default to NULL.